### DISCUSSION TITLE
[Config Support]:

### DISCUSSION THREAD

**Discussion by diegargon** [blakeblackshear/frigate#13743] [Config Support]
### Describe the problem you are having

Hello.

I already use frigate one year ago without problems now when i try install again this error appears.

### Version

0.14.1 and stable

### Frigate config file

```yaml
mqtt:
  enabled: false
  host: mqtt.XXX
  user: mqtt
  password: XXXX
cameras:
  CamArbol:
    enabled: false
```

### Relevant Frigate log output

```shell
Can't create container from addon_ccab4aaf_frigate-beta: 400 Client Error for http+docker://localhost/v1.41/containers/create?name=addon_ccab4aaf_frigate-beta: Bad Request ("invalid CapAdd: unknown capability: "CAP_PERFMON"")
```

### Relevant go2rtc log output

```shell
01:03:32.032 INF go2rtc version 1.6.2 linux/amd64
01:03:32.038 INF [api] listen addr=0.0.0.0:1984
01:03:32.039 INF [rtsp] listen addr=0.0.0.0:8554
01:03:32.042 INF [webrtc] listen addr=0.0.0.0:8555/tcp
01:03:32.046 INF [srtp] listen addr=0.0.0.0:8443
exit with signal: terminated
20:19:09.653 INF go2rtc version 1.6.2 linux/amd64
20:19:09.659 INF [api] listen addr=0.0.0.0:1984
20:19:09.660 INF [rtsp] listen addr=0.0.0.0:8554
20:19:09.662 INF [webrtc] listen addr=0.0.0.0:8555/tcp
20:19:09.664 INF [srtp] listen addr=0.0.0.0:8443
exit with signal: terminated
20:19:28.005 INF go2rtc version 1.6.2 linux/amd64
20:19:28.006 INF [api] listen addr=0.0.0.0:1984
20:19:28.006 INF [rtsp] listen addr=0.0.0.0:8554
20:19:28.006 INF [webrtc] listen addr=0.0.0.0:8555/tcp
20:19:28.009 INF [srtp] listen addr=0.0.0.0:8443
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

CPU (no coral)

**Comment by NickM-27** [comment]
this usually indicates an out of date supervisor system. See https://github.com/blakeblackshear/frigate/issues/9527

**Comment by diegargon** [comment]
will check that thread this week but i have update HA to the last version.

Versión | core-2024.9.1
-- | --
Tipo de instalación | Home Assistant Supervised
Desarrollo | false
Supervisor | true
Docker | true
Usuario | root
Entorno virtual | false
Versión de Python | 3.12.4
Familia del sistema operativo | Linux
Versión del sistema operativo | 5.10.0-14-amd64
Arquitectura de la CPU | x86_64
Zona horaria | Europe/Madrid
Directorio de configuración | /config



Perhaps a  kernel problem? but is supposed update with system since is supervised. 

I don't want update the kernel manually and break the "official release"

**Comment by diegargon** [comment]
I do not know at what time I assumed that the version "supervised" update the base system, and it seems that it is not like that

**Comment by diegargon** [comment]
Upgrade OS fix it. Thx
